### DISCUSSION TITLE
Reverse Engineer Whisper for TTS

### DISCUSSION THREAD

**Discussion by tiwale01** [openai/whisper#748] [Ideas]
Is it possible to reverse engineer whisper for TTS? Since whisper recognizes more languages than TTS engines, it would be nice for languages with no TTS. Did I ask the right question?

**Comment by dilmerv** [comment]
Looking for the same answer right now and yes you did ask the right question. I am currently using Wit.ai with Meta Voice SDK for VR development but I am finding that it takes too long to convert text to speech and it also has a max of 280 characters per request, it would be great to have a near to real-time solution such as Whisper to do something similar.

**Reply by ankitsharma07** [reply]
I am currently using gTTS library for TTS. It also have a characters limit per request and the voice sounds too robotic. An open source TTS engine which is really good would be nice to have functionality.

**Reply by ADD-eNavarro** [reply]
What about [coqui](https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html)?
